packages/components/src/templates/next/components/complex/Hero/HeroLargeImage/HeroLargeImage.tsx

```diff
@@ -0,0 +1,72 @@
+import type { HeroProps } from "~/interfaces/complex/Hero"
+import { getReferenceLinkHref, isExternalUrl } from "~/utils"
+import { ComponentContent } from "../../../internal/customCssClass"
+import { LinkButton } from "../../../internal/LinkButton/LinkButton"
+import { ImageContainer } from "./ImageContainer"
+
+export const HeroLargeImage = ({
+  title,
+  subtitle,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+  backgroundUrl,
+  site,
+  LinkComponent,
+}: HeroProps) => {
+  const backgroundSrc =
+    isExternalUrl(backgroundUrl) || site.assetsBaseUrl === undefined
+      ? backgroundUrl
+      : `${site.assetsBaseUrl}${backgroundUrl}`
+
+  return (
+    <section className="flex w-full flex-col">
+      {/* Text and button container */}
+      <div
+        className={`mx-auto flex w-full flex-col gap-6 px-6 pb-12 pt-10 md:gap-9 lg:pb-16 lg:pt-12 ${ComponentContent}`}
+      >
+        {/* Text container */}
+        <div className="flex flex-col items-center gap-5 text-base-content-strong sm:items-start md:gap-6 md:text-center lg:mx-auto lg:max-w-[66.67%]">
+          <h1 className="prose-display-xl w-full">{title}</h1>
+          {subtitle && (
+            <p className="prose-title-lg-regular w-full">{subtitle}</p>
+          )}
+        </div>
+        {/* Button container */}
+        {buttonLabel && buttonUrl && (
+          <div className="flex flex-col items-start justify-center gap-x-5 gap-y-4 md:flex-row md:items-center">
+            <LinkButton
+              href={getReferenceLinkHref(
+                buttonUrl,
+                site.siteMap,
+                site.assetsBaseUrl,
+              )}
+              size="lg"
+              LinkComponent={LinkComponent}
+              isWithFocusVisibleHighlight
+            >
+              {buttonLabel}
+            </LinkButton>
+            {secondaryButtonLabel && secondaryButtonUrl && (
+              <LinkButton
+                variant="outline"
+                size="lg"
+                href={getReferenceLinkHref(
+                  secondaryButtonUrl,
+                  site.siteMap,
+                  site.assetsBaseUrl,
+                )}
+                LinkComponent={LinkComponent}
+                isWithFocusVisibleHighlight
+              >
+                {secondaryButtonLabel}
+              </LinkButton>
+            )}
+          </div>
+        )}
+      </div>
+      <ImageContainer imageSrc={backgroundSrc} imageAlt={title} />
```

Comment:
[sensecheck] this isnt being eagerly loaded because it's below the fold right?

Reply:
it is! it renders <ImageClient> under the hood with a hardcoded `lazyLoading={false}` - safe assumption here is that hero will always be the first block

Reply:
ah oops, this is `ImageContainer` and not `ImageClient`